mmcv/cnn/resnet.py

```diff
@@ -255,12 +256,13 @@ def __init__(self,
                 dilation=dilation,
                 style=self.style,
                 with_cp=with_cp)
-            self.inplanes = planes * block.expansion
+            self.inplanes = planes * block.expansion  # type: ignore
```

Comment:
It could be better to pre-define the type of `self.inplanes` just like `self.inplanes: Union[int, xxx]`